CONTRIBUTING.rst

```diff
@@ -0,0 +1,177 @@
+
+.. tip::
+    Use the templates included below for further guidance. 
+
+To learn more about documentation content types, see the `Diátaxis Framework <https://diataxis.fr/>`_.
+
+
+**What is the structure of DevPortal?**
+
+You can see the structure in the left-hand navigation menu. 
+It is driven by a plugin called `Sphinx external TOC <https://sphinx-external-toc.readthedocs.io/en/latest/intro.html>`_. 
+
+You can find the current structure in ``_toc.yml``.
+
+If you have an article about any of the Aiven tools (API, CLI, console, Terraform provider, Kubernetes operator), place it in the **Aiven tools** section.
+If you have an article about any specific Aiven product (M3, Kafka, Grafana, etc.), nest it in the product-specific section. 
+If you have an article with general reference information, place it in the **Resources** section. 
+
+
+How can I contribute?
+---------------------
+
+Reporting bugs, and suggesting enhancements
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you notice something is off (anything - from typos to completely incorrect instructions) or missing, consider opening an issue. 
+
+.. tip::
+    Before you open an issue, check all the existing issues for the DevPortal to make sure you're not duplicating it. 
+
+
+Contributing via pull requests (PRs)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you have your own article to add or you want to modify an existing article yourself, here's what you do:
+
+1. Fork the repo.
+2. Make your update.
+3. Open a pull request (make sure you use a helpful PR and commit message). 
+4. Submit your PR, and get it reviewed (one reviewer is required). 
+
+We'll merge it!
+
+.. note::
+    Our build runs several checks on your updates - from linting to spell check to broken links to help you commit good content. Look at the **Checks** section in your pull request to make sure everything passes. If anything fails, check the error details, or reach out to us, and we'll help you. 
+
+
+Style tips
+==========
+
+If you're writing content, here are some high-level tips that can be useful:
+
+- Decide what type of content you want to contribute (check the content model above), and use our content type templates - for concept, task, reference and tutorial.
+
+- Write a good title (start with a verb for tasks, and noun for concepts, or use one of the templates below to guide you). 
+
+- Note down your main ideas, and decide what formatting will best support these (use the content templates):
+    - Ordered lists - for tasks
+    - Paragraphs - for definitions and explanations
+    - Lists and tables - for reference/options
+
+- Highlight information by using the following elements:
+    - Tips - for interesting facts about parts of your content/alternative ways of doing something, and shortcuts
+    - Warnings - for important facts readers need to know before proceeding
```

Comment:
I wonder is it worth changing this to "Important" or something else? It might be more of a general documentation standard, but I'm use to rather strict categorization for warning, caution, and danger notifications, where warnings are usually reserved for situations that could result in serious physical injury or death :)

Reply:
Agree!